tests/Oryx.RuntimeImage.Tests/Php-fpm/PhpFpmImageTest.cs

```diff
@@ -0,0 +1,122 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.Tests.Common;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.RuntimeImage.Tests
```

Comment:
Can you write some integrations tests too?

Reply:
yes, that will be next step, integration tests will require adjusting build and release pipeline , and I don't want to address those in this PR. I have planned to create a separate PR with workitem for that.